### DISCUSSION TITLE
can you provide access to Intelligent-VMS-Visitor-Management-System ' s .env file

### DISCUSSION THREAD

**Discussion by Liyansu** [COS301-SE-2022/Intelligent-VMS-Visitor-Management-System-#324] [General]
I am liyansu i read your documentation
Furthermore the backend system relies on sensitive data that is stored in the .env file and is not part of the public repo for security and confidentiality reasons. Contact one of the firestorm team members to give access to the .env file.
please provide credentials to access

**Comment by HandsomeBiba** [comment]
Me too

**Reply by dirtysnipe-ux** [reply]
Also requesting this file please.
